cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -423,26 +419,31 @@ instance ( ConvertRawHash blk
             renderPointAsPhrase (AF.headPoint c) <> ", slots " <>
             Text.intercalate ", " (map (renderPoint . headerPoint) hdrs)
           ChainDB.UpdateLedgerDbTraceEvent
-            (StartedPushingBlockToTheLedgerDb (Pushing currentBlock) (PushGoal pushGoal)) ->
+            (StartedPushingBlockToTheLedgerDb (PushStart pushOrigin) (PushGoal pushGoal) (Pushing currentBlock)) ->
             "About to push " <> renderRealPoint currentBlock <>
-            " to ledger DB. Attempting to push blocks until " <> renderRealPoint pushGoal
+            " to ledger DB. Attempting to push blocks" <>
+            " from " <> renderRealPoint pushOrigin <>
+            " until " <> renderRealPoint pushGoal
         ChainDB.AddedBlockToVolatileDB pt _ _ ->
           "Chain added block " <> renderRealPointAsPhrase pt
         ChainDB.ChainSelectionForFutureBlock pt ->
           "Chain selection run for block previously from future: " <> renderRealPointAsPhrase pt
       ChainDB.TraceLedgerReplayEvent ev -> case ev of
         LedgerDB.ReplayFromGenesis _replayTo ->
           "Replaying ledger from genesis"
-        LedgerDB.ReplayFromSnapshot snap tip' _replayTo ->
+        LedgerDB.ReplayFromSnapshot snap tip' (LedgerDB.ReplayStart replayFrom) (LedgerDB.ReplayGoal replayTo) ->
           "Replaying ledger from snapshot " <> showT snap <> " at " <>
-            renderRealPointAsPhrase tip'
-        LedgerDB.ReplayedBlock pt _ledgerEvents replayTo ->
+            renderRealPointAsPhrase tip' <> ".  Pushing"
```

Comment:
```suggestion
            renderRealPointAsPhrase tip' <> ".  Replaying"
```

Reply:
@EncodePanda @Jasagredo do you agree with this suggestion? :point_up:

Reply:
well, it does not fired during replaying but chain selection and pushing block to the ledger, we have separate events for replaying